integration/quarks_secret_test.go

```diff
@@ -185,4 +185,28 @@ var _ = Describe("QuarksSecret", func() {
 			})
 		})
 	})
+
+	It("generates a basic auth secret with a username and password and deletes it when being deleted", func() {
+		// Create an QuarksSecret
```

Comment:
I also like using `By("creating a QuarksSecret")` instead of comments. Your choice.

Reply:
I think my teammates are trying to match the structure of the other test blocks in the same test file (`It` with comments). Perhaps we could leave this for now, and we could change all test blocks into `By` structure in the future if you prefer?